stable_baselines3/common/env_util.py

```diff
@@ -109,6 +112,7 @@ def make_atari_env(
     env_kwargs: Optional[Dict[str, Any]] = None,
     vec_env_cls: Optional[Union[DummyVecEnv, SubprocVecEnv]] = None,
     vec_env_kwargs: Optional[Dict[str, Any]] = None,
+    monitor_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
This argument is not used anywhere in this function (I guess should be passed through to `make_vec_env` later)?

Reply:
nice catch, yes